lib/ExecutionControlEpic/DiagnosticsFeature/Presenters/DiagnosticDetailsFlow.js

```diff
@@ -79,27 +80,37 @@ export default class DiagnosticDetailsFlow extends React.Component<
         ),
       List(),
     );
-    return (
-      <Flow
-        innerRef={ref => {
-          this.contain = ref;
-        }}
-      >
-        {diagnostics.map((diagnostic, i) => (
-          <Detail
-            key={i}
-            innerRef={e => {
-              i == diagnostics.size - 1 ? (this.list = e) : null;
-            }}
-          >
-            <DiagnosticDetails
-              diagnostic={diagnostic}
-              jumpTo={this.props.onJumpTo}
-            />
-          </Detail>
-        ))}
-      </Flow>
-    );
+    if (diagnostics === List()) {
```

Comment:
You should use diagnostics.size === 0 instead. (see [List#size](https://facebook.github.io/immutable-js/docs/#/List/size)).